packages/rrweb/src/record/index.ts

```diff
@@ -41,8 +41,10 @@ import {
 } from './error-handler';
 
 let wrappedEmit!: (e: eventWithoutTime, isCheckout?: boolean) => void;
-
 let takeFullSnapshot!: (isCheckout?: boolean) => void;
+takeFullSnapshot = () => {
```

Comment:
why not just combine this with the definition above?

```
let takeFullSnapshot!: (isCheckout?: boolean) => void = () => {};
```

Reply:
lol because I don't know TS very well and couldn't figure out the syntax.. I'll update it